rr_arduino/imu.ino

```diff
@@ -0,0 +1,131 @@
+#include <Wire.h>
+#include <Adafruit_Sensor.h>
+#include <Adafruit_BNO055.h>
+#include <utility/imumaths.h>
+#include <ros.h>
+#include <std_msgs/Int8.h>
+#include <std_msgs/Float32.h>
+#include <std_msgs/Float32MultiArray.h>
+
+#define TEST_OUTPUT
+#define IMU_BAUD_RATE       38400
+
+
+//creating message objects
+std_msgs::Float32MultiArray imu_msg;
+sensory_msgs::MagneticField magnetic_msg;
```

Comment:
the contents of this file should be integrated into the main arduino code

Reply:
Because commands of the same type exist in the arduino code, I assuming this is what should be integrated.
-publisher commands
-advertise commands
-message commands
-ROS node initialization